### DISCUSSION TITLE
Unable to downgrade 16 core machine

### DISCUSSION THREAD

**Discussion by herman5** [community/community#45298] [Codespaces]
### Select Topic Area

Bug

### Body

I'm attempting to downgrade a codespace machine type from 16 cores to 4 cores. However, the operation is hanging (downgrading and upgrading are normally  fast operations) and I'm unable to downgrade it to any other machine type (have attempted 2, 4, and 8 core machine types). 

The issue is further exasperated by giving no feedback that the operation failed. I unknowingly spent more time on a 16 core machine having thought it was successfully downgraded to 4 cores - consuming all my allotted compute time for the month.

I've attempted to reproduce this by downgrading 16 core machine on other repos as well as new machines for the same repo that is having the issue. These however have all been successful.

**Comment by dmilburn** [comment]
Hi @herman5, i'm sorry for the inconvenience and thank you for reaching out. A few questions:

- I know you said you haven't been able to reproduce this on other repos. Have you tried creating a new codespace on the same repo and seeing whether you have the same problem?
- Could you share the name of the codespace or repo where this was occurring?

**Reply by herman5** [reply]
The problem is specific to this single repo when it had large files that were unable to be committed due to size limitations. When I migrated the large files I was able to downgrade the machine.
